src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreWorker.cs

```diff
@@ -222,7 +216,12 @@ private void SolutionEvents_BeforeClosing()
         private void SolutionEvents_AfterClosing()
         {
             Reset();
-            ErrorListProvider.Tasks.Clear();
+
+            // Clear warnings/errors from nuget
+            if (_errorListTableDataSource.IsValueCreated)
```

Comment:
There is no reason to create it just to noop. If it isn't created then it has never been used for logging, and doesn't need focus.